src/validation.h

```diff
@@ -115,6 +115,21 @@ void PruneBlockFilesManual(Chainstate& active_chainstate, int nManualPruneHeight
 
 /**
 * Validation result for a single transaction mempool acceptance.
```

Comment:
ah, this text line is old; can it be updated to note that this table is for package contexts? otherwise the "txid in mempool" and "wtxid in mempool" differ a bit for the `INVALID` cases, since they cover the MEMPOOL_ENTRY and DIFFERENT_WITNESS cases for ATMP cases

Reply:
Thanks, changed/added comment in validation.h